azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositeLocalOptions.java

```diff
@@ -89,4 +91,62 @@ public CallCompositeLocalOptions setSetupScreenViewData(
         this.setupScreenViewData = setupScreenViewData;
         return this;
     }
+
+    /**
+     * Get the {@link Boolean}.
```

Comment:
I think in this case {@link Boolean} does not provide much information. 
How about "Get skip setup screen value."

Reply:
Same for all new fields in this class